api/agent/drivers/docker/docker_client.go

```diff
@@ -308,6 +311,19 @@ func (d *dockerWrap) LoadImages(ctx context.Context, filePath string) error {
 	})
 }
 
+func (d *dockerWrap) ListImages(ctx context.Context) (imgs []docker.APIImages, err error) {
+	ctx, closer := makeTracker(ctx, "list_docker_images")
```

Comment:
can we do `docker_list_images` to be consistent with the other stats in this file please? this makes eg summing over `docker_*` ops easy, aside from consistency benefits for being easy to find